project/src/attachments/Camera.gd

```diff
@@ -37,6 +37,15 @@ var fps = 0
 var vflip: bool = false
 var hflip: bool = false
 
+signal cam_locked
```

Comment:
This whole file is the camera **attachment** not the actual camera that is used by the user